web/virtual_lab/static/virtual_lab/js/physics_projectile.js

```
@@ -0,0 +1,412 @@
+  const yReadout = document.getElementById("y-readout");
+  const vxReadout = document.getElementById("vx-readout");
+  const vyReadout = document.getElementById("vy-readout");
+
+  let g = Number.parseFloat(gravitySlider.value);
+  let windAccel = Number.parseFloat(windSlider.value);
+
+  let originY = canvas.height - 10;
+  let pixelsPerMeter = 10;
+
+  let v0 = 0, thetaRad = 0, vx = 0, vy0 = 0;
+  let maxRange = 0, maxHeight = 0;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)